expression/functions_math.go

```diff
@@ -209,24 +209,13 @@ func roundToDecimal(input interface{}, decimalPlaces interface{}) (interface{},
 	return rounded / math.Pow(10, float64(intDecimalPlaces)), nil
 }
 
-func convertAsFloat(value interface{}) float64 {
-	switch v := value.(type) {
-	case int, int32, int64:
-		return float64(v.(int))
-	case float32, float64:
-		return v.(float64)
-	default:
-		return 0
-	}
-}
-
-func safeDivide(divider interface{}, dividende interface{}) float64 {
-	floatDivider := convertAsFloat(divider)
-	floatDividende := convertAsFloat(dividende)
+func safeDivide(dividende interface{}, divider interface{}) float64 {
```

Comment:
```suggestion
func safeDivide(dividend interface{}, divisor interface{}) float64 {
```
In english it is called:

_What is being divided is called the dividend, which is divided by the divisor, and the result is called the quotient._

Reply:
fixed in last commit